src/module.h

```diff
@@ -57,6 +57,7 @@ do {                                            \
   RedisModule_ReplyWithStringBuffer(ctx, literal, sizeof(literal) - 1)
 
 #define SEARCH_ACL_CATEGORY "search"
+#define SEARCH_ACL_INTERNAL_CATEGORY "search_internal"
```

Comment:
Can we name the internal category with a leading underscore? (as internal commands)

Reply:
Yes that makes sense 👌